services/brig/src/Brig/API/Internal.hs

```diff
@@ -302,11 +307,36 @@ createUserNoVerify uData = do
   let uid = userId usr
   let eac = createdEmailActivation result
   let pac = createdPhoneActivation result
-  for_ (catMaybes [eac, pac]) $ \adata ->
-    let key = ActivateKey $ activationKey adata
-        code = activationCode adata
-     in API.activate key code (Just uid) !>> actError
+
+  if isNewUserInvitedViaScim uData
+    then do
+      case (eac, newUserEmail uData) of
+        (Just adata, Just email) -> do
+          void $
+            lift $
+              sendActivationEmail
```

Comment:
```suggestion
        (Just adata, Just email) -> do
          void . lift $ sendActivationEmail
```

not sure if ormolu won't add a newline after `$`, but that's at least one less than here.  :)